Dockerfile

```diff
@@ -21,7 +20,6 @@ RUN apt-get update && \
   && rm -rf /var/lib/apt/lists/*
 
 RUN gem install bundler --version "${BUNDLER_VERSION}"
-RUN npm install -g eslint@${ESLINT_VERSION}
```

Comment:
What do you think about leaving the default eslint runner (globally installed) and allowing the user to override with the `pronto_eslint_npm` config file? That might be less of a breaking change (since people could be relying on this—we are at Adwerx) and the user could always decide to override?

Reply:
@jbielick I was thinking about that, but using global `eslint` for some reason does not pick up local `node_modules`. Only using local `eslint` works with local `node_modules`

Reply:
🤔 Are you sure? This is what we've got and it's been working well for a while. We have no pronto-eslintnpm config.

```yaml
    steps:
      - uses: actions/checkout@v1
      - uses: actions/setup-node@v1
        with:
          node-version: '10.x'
      - run: yarn install --ignore-optional --ignore-scripts --frozen-lockfile --non-interactive
      - uses: adwerx/pronto-ruby@v2.5
        with:
          runners: >-
            rubocop rails_schema yamllint poper erb_lint eslint_npm
```

Reply:
At least it is what I saw on our local repository. I will double-check and get back to you with different outputs.

Reply:
@Saicheg did you get a chance to test this? Don't want this PR to get stale.

Reply:
@jbielick hey, it's on my list, just got busy with some other things. No worries - will get back to you with results!